goonstation.dme

```diff
@@ -49,6 +49,31 @@ var/datum/preMapLoad/preMapLoad = new
 #include "code\__secret.dm"
 
 // BEGIN_INCLUDE
+#include "_stdlib\__build.dm"
```

Comment:
delet this

Reply:
Right

Reply:
Okay, it wasn't just the one shown in the snippet. Removed the requested lines!